packages/instant/src/util/analytics.ts

```diff
@@ -0,0 +1,49 @@
+import { ObjectMap } from '@0x/types';
+
+import { heapUtil } from './heap';
+
+enum EventNames {
```

Comment:
We maybe want something like name spacing for events so we can have events that are like `Wallet-Dropdown-Opened`  and we can enforce consistency for "open" events, that should make navigating the data easier down the road